Soccar [Game-Server]/src/nl/soccar/gameserver/SessionWrapper.java

```diff
@@ -81,10 +81,6 @@ public void leaveSession(Player player) {
 
         PlayerLeftSessionMessage m = new PlayerLeftSessionMessage(player.getUsername(), team.getTeamColour());
         room.getAllPlayers().stream().map(server::getConnectionFromPlayer).forEach(c -> c.send(m));
-
-        if (room.getOccupancy() <= 0) {
-            controller.destroySession(session);
-        }
     }
 
     public void startGame() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)